src/renderers/shaders/ShaderChunk/cube_uv_reflection_fragment.glsl

```diff
@@ -3,126 +3,126 @@
 const float cubeUV_textureSize = 1024.0;
 
 int getFaceFromDirection(vec3 direction) {
-    vec3 absDirection = abs(direction);
-    int face = -1;
-    if( absDirection.x > absDirection.z ) {
-        if(absDirection.x > absDirection.y )
-            face = direction.x > 0.0 ? 0 : 3;
-        else
-            face = direction.y > 0.0 ? 1 : 4;
-    }
-    else {
-        if(absDirection.z > absDirection.y )
-            face = direction.z > 0.0 ? 2 : 5;
-        else
-            face = direction.y > 0.0 ? 1 : 4;
-    }
-    return face;
+		vec3 absDirection = abs(direction);
```

Comment:
This file is now using double tabs instead of single tabs for indentation.
With that fixed it should be good to go! :+1: